cpp/doxygen/developer_guide/TESTING.md

```diff
@@ -65,7 +68,7 @@ Example:
 
 ## Typed Tests
```

Comment:
We also support value-parameterized test now:
https://github.com/rapidsai/cuspatial/blob/d044dc42fb6d5d5045b8745036014d26ac09a6ab/cpp/include/cuspatial_test/base_fixture.hpp#L80

Reply:
Separate PR (same as above).